src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
```

Comment:
Seems like we should have a constructor to create a range and then have:
```
LowerRange(LIR::AsRange(firstNode, lastNode));
```
or maybe have callers do that. But I don't see that functionality currently.